Xamarin.Forms.Platform.iOS/VisualElementRenderer.cs

```diff
@@ -73,6 +73,16 @@ protected bool AutoTrack
 			}
 		}
 
+		public static void RegisterEffect(Effect effect, UIView container, UIView control = null)
```

Comment:
Since `RegisterEffect` is the same across all renderers, this can be refactored into its own method.